WeartGloves/src/WeArtGlove.cpp

```diff
@@ -0,0 +1,1598 @@
+﻿/**
+ * @file WeArtGlove.cpp
+ * @authors Ehsan Ranjbari <ehsan.ranjbari@iit.it>
+ * @copyright 2022 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2022
+ */
+
+#include <WeArtGlove.h>
+#include <WeArtGloveLib/WeArtGloveLib.h>
+
+#include <yarp/os/Bottle.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Time.h>
+
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/RpcServer.h>
+#include <yarp/os/ResourceFinder.h>
+
+#include <assert.h>
+#include <cmath>
+#include <map>
+#include <mutex>
+#include <stdio.h>
+#include <string>
+#include <thread>
+#include <vector>
+#include <algorithm>
+#include <cstdlib>
+
+#include <Eigen/Dense>
+#include <Eigen/Core>
+
+// iDynTree headers
+#include <iDynTree/Model/FreeFloatingState.h>
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/Core/EigenHelpers.h>
```

Comment:
```suggestion
#include <iDynTree/FreeFloatingState.h>
#include <iDynTree/KinDynComputations.h>
#include <iDynTree/ModelLoader.h>
#include <iDynTree/EigenHelpers.h>
```

I guess this style of include is triggering some iDynTree warnings, perhaps we can solve them like this?